src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -75,6 +75,11 @@ public async Task Handle(MigrateRepoCommandArgs args)
             {
                 args.ArchivePath = await DownloadArchive(exportId);
             }
+
+            if (!args.ShouldDownloadArchive() && args.ShouldUploadArchive())
```

Comment:
So this definitely would prompt a user to pull in the correct arguments. I wonder if the base premise of users running this command in BBS is something we should consider first

Reply:
We definitely consider the ergonomics of the command - I do think the default is "wrong"! - but changing the behaviour is likely to be a breaking change for existing users, so we need to consider that carefully.

Given that, I think I'd choose to add a band-aid for now to stop the bleeding, and then take a step back and think about the right long-term solution.

Reply:
Just so you know, you wouldn't get this log line in the case you ran into because you didn't specify a target GitHub organization and repo. But you could get the other log line you commented on below.